src/DDM.jl

```diff
@@ -381,67 +383,325 @@ sampler(rng::AbstractRNG, d::DDM) = rand(rng::AbstractRNG, d::DDM)
 # """
 #     RatcliffDDM 
```

Comment:
Given how long this file is, would you be willing to move this commented code into a separate file?

Reply:
I will create a new file called: `RatcliffDDM .jl`